warehouse/query-core/src/main/java/datawave/query/index/lookup/TupleToRange.java

```diff
@@ -1,5 +1,6 @@
 package datawave.query.index.lookup;
 
+import java.util.Collection;
```

Comment:
Similar deal, this could be removed if you like.

Reply:
Good catch, thanks!